smart_aquarium/lib/ui/registerPage.dart

```diff
@@ -102,108 +104,156 @@ class RegisterPage extends StatelessWidget {
-                  labelText: 'Nazwa',
-                  labelStyle: TextStyle(color: Color(0xFF828282)),
-                  prefixIcon: Icon(Icons.person, color: Color(0xFF828282)),
-                  border: OutlineInputBorder(
-                    borderSide: BorderSide(color: Color(0xFF828282)),
-                  ),
-                ),
-              ),
-              const SizedBox(height: 20.0),
-              TextField(
-                controller: _emailController,
-                style: const TextStyle(color: Color(0xFF828282)),
-                decoration: const InputDecoration(
-                  labelText: 'Email',
-                  labelStyle: TextStyle(color: Color(0xFF828282)),
-                  prefixIcon: Icon(Icons.email, color: Color(0xFF828282)),
-                  border: OutlineInputBorder(
-                    borderSide: BorderSide(color: Color(0xFF828282)),
-                  ),
-                ),
-              ),
-              const SizedBox(height: 20.0),
-              TextField(
-                controller: _passwordController,
-                style: const TextStyle(color: Color(0xFF828282)),
-                obscureText: true,
-                decoration: const InputDecoration(
-                  labelText: 'Hasło',
-                  labelStyle: TextStyle(color: Color(0xFF828282)),
-                  prefixIcon: Icon(Icons.lock, color: Color(0xFF828282)),
-                  border: OutlineInputBorder(
-                    borderSide: BorderSide(color: Color(0xFF828282)),
-                  ),
+        child: LayoutBuilder(
+          builder: (BuildContext context, BoxConstraints constraints) {
+            return SingleChildScrollView(
+              child: ConstrainedBox(
+                constraints: BoxConstraints(
+                  minHeight: constraints.maxHeight,
                 ),
-              ),
-              const SizedBox(height: 20.0),
-              Container(
-                height: 50.0,
-                width: double.infinity,
-                decoration: BoxDecoration(
-                  color: Colors.transparent,
-                  borderRadius: BorderRadius.circular(10.0),
-                ),
-                child: ElevatedButton(
-                  onPressed: () => _register(context),
-                  style: ElevatedButton.styleFrom(
-                    backgroundColor: Colors.blue,
-                    elevation: 0,
-                  ),
-                  child: const Text(
-                    'Stwórz konto',
-                    style: TextStyle(
-                        fontSize: 20.0,
-                        color: Colors.white), // Rozmiar tekstu w przycisku
+                child: Padding(
+                  padding: const EdgeInsets.all(20.0),
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: <Widget>[
+                      Image.asset(
+                        'assets/login_logo.png',
```

Comment:
This is more of a comment to the other asset, but since we're using pngs anyway wouldn't it be better to use transparent image background instead of white?

Reply:
It was a problem with image, I changed it to another with transparent background